content/rest-api/builds.md

```diff
@@ -19,14 +19,15 @@ APIs for managing builds are currently available for developers to preview. Duri
 
 #### Parameters
 
-| **Name**      | **Type** | **Description** |
-| ------------- | -------- | --------------- |
-| `appId`       | `string` | **Required.** The application identifier. |
-| `workflowId`  | `string` | **Required.** The workflow identifier as specified in YAML file. |
-| `branch`      | `string` | Optional. The branch name. Either `branch` or `tag` is **required**. |
-| `tag`         | `string` | Optional. The tag name. Either `branch` or `tag` is **required**. |
-| `environment` | `object` | Optional. Specify environment variables, variable groups, and software versions to override or define in workflow settings. | 
-| `labels`      | `list`   | Optional. Specify labels to be included for the build in addition to existing labels. |
+| **Name**      | **Type** | **Description**                                                                                                                                         |
+|---------------| -------- |---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `appId`       | `string` | **Required.** The application identifier.                                                                                                               |
+| `workflowId`  | `string` | **Required.** The workflow identifier as specified in YAML file.                                                                                        |
+| `branch`      | `string` | Optional. The branch name. Either `branch` or `tag` is **required**.                                                                                    |
+| `tag`         | `string` | Optional. The tag name. Either `branch` or `tag` is **required**.                                                                                       |
+| `environment` | `object` | Optional. Specify environment variables, variable groups, and software versions to override or define in workflow settings.                             |
+| `inputs`      | `object` | Optional. Specify values for build inputs. Read more about from [here](/knowledge-codemagic/build-inputs/#specify-inputs-when-starting-builds-with-api) |
```

Comment:
imho better to revert this changes. There is no schema information or example.

Reply:
I've added yaml schema. Should we still remove this?

Reply:
yes, better to remove. it's not about yaml but request schema